src/Blocks/BlockCrops.h

```diff
@@ -122,8 +149,30 @@ class cBlockCropsHandler:
 		UNUSED(a_Meta);
 		return 7;
 	}
-} ;
 
 
 
 
+
+private:
+
+	/**
+		Generate a random number using a binomial distribution
+		See https://minecraft.gamepedia.com/Fortune#Seeds
+	*/
+	int GenerateAmountSeedDrop(int a_Min, int a_Count, double a_Probability, MTRand &a_Rand)
+	{
+		int SeedCount = a_Min;
+		for (int i(0); i < a_Count; i++)
```

Comment:
```suggestion
		for (int i = 0; i < a_Count; i++)
```

This method of initialisation is odd and unusual, for consistency of style please use the normal method.